lib/common/validation.js

```diff
@@ -2122,3 +2242,10 @@ function validVmField(field) {
         field === 'role_tags';
 }
 exports.validVmField = validVmField;
+
+function isSortOrderDesc(order) {
```

Comment:
I would name this `isSortOrderDescending()` instead, to be a bit more clear to readers (at the call site) about what it does.